build.gradle.kts

```diff
@@ -75,6 +75,13 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
 
+    // mysql
+    implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+    implementation("org.liquibase:liquibase-core")
+    runtimeOnly("mysql:mysql-connector-java")
+    runtimeOnly("software.aws.rds:aws-mysql-jdbc:1.1.1")
```

Comment:
Adding the AWS MySql deps here will trigger a load of owasp dependency scanner false positives! 😢 
Please can you add the relevant suppression to the owasp suppressions file (copy the relevant bit from VCA or RCA's version of the file 👍 )

Reply:
Suppressed 👍